benchalerts/README.md

```diff
@@ -0,0 +1,110 @@
+# benchalerts
+
+A package to facilitate automated alerting based on Conbench data.
+
+## Overview
+
+This package is intended to make the following steps easier in CI. Before these steps,
+it is assumed that an execution environment has performed a run of benchmarks and
+submitted the results to Conbench.
+
+- Hit the Conbench API to understand if there were any:
```

Comment:
For the future: 

I'd love to see a summary for the method/algorithm applied here so far.

Reply:
Great idea! Right now, it basically just hits 
`/api/compare/runs/{baseline}...{contender}/?threshold_z={threshold_z}`
 for each contender run associated with the given commit. All the logic for determining what the baseline run is and how to define the z-score with which to threshold is on the server's side. I can put this in the README.